communication.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto2";
```

Comment:
the proto file name can be more specific.

Reply:
Ok 🙂